I'm sorry, but an uncaught exception occurred.

While running game code:
  File "renpy/common/00start.rpy", line 192, in script
    python:
  File "renpy/common/00start.rpy", line 193, in <module>
    renpy.execute_default_statement(True)
  File "game/lifePlayer.rpy", line 48, in set_default
    default user = LifePlayer()
  File "game/lifePlayer.rpy", line 48, in <module>
    default user = LifePlayer()
  File "game/lifePlayer.rpy", line 8, in __init__
    self.race_bias = Races[self.race][self.gender]
NameError: name 'Races' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "renpy/common/00start.rpy", line 192, in script
    python:
  File "/Users/Arnav1/Documents/renpy-8.0.3-sdk/renpy/ast.py", line 1131, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "/Users/Arnav1/Documents/renpy-8.0.3-sdk/renpy/python.py", line 1061, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "renpy/common/00start.rpy", line 193, in <module>
    renpy.execute_default_statement(True)
  File "/Users/Arnav1/Documents/renpy-8.0.3-sdk/renpy/exports.py", line 3797, in execute_default_statement
    i.set_default(start)
  File "game/lifePlayer.rpy", line 48, in set_default
    default user = LifePlayer()
  File "/Users/Arnav1/Documents/renpy-8.0.3-sdk/renpy/python.py", line 1085, in py_eval_bytecode
    return eval(bytecode, globals, locals)
  File "game/lifePlayer.rpy", line 48, in <module>
    default user = LifePlayer()
  File "game/lifePlayer.rpy", line 8, in __init__
    self.race_bias = Races[self.race][self.gender]
NameError: name 'Races' is not defined

macOS-10.16-x86_64-i386-64bit x86_64
Ren'Py 8.0.3.22090809
CS492_Project 1.0
Sun Apr  9 16:01:09 2023
